File: 1.FDP/2° Competencia/Java/Basicos/Ejercicio3.java
import java.util.Scanner;

public class Ejercicio3 {
	public static void main (String[] args){
		Scanner scan= new Scanner(System.in);
		//
		System.out.println("Calcular el area de un triangulo.\n");
		double Base, Altura, Area;
		System.out.println("Base: ");
		Base= scan.nextDouble();
		System.out.println("Area: ");
		Altura= scan.nextDouble();
		Area= (Base*Altura)/2;
		System.out.println("Area= " + Area);
		//
	}
}

File: 1.FDP/2° Competencia/C++/Basicos/Ejercicio3.cpp
#include <iostream>
using namespace std;

int main(){
	cout << "Calcular el area de un triangulo.\n";
	double Altura, Base, Area;
	cout << "Altura: ";
	cin >> Altura;
	cout << "Base: ";
	cin >> Base;
	Area= (Base*Altura)/2;
	cout << "Area = " << Area << "\n";
	return 0;
}